specification/esql/query/QueryRequest.ts

```diff
@@ -26,6 +26,7 @@ import { EsqlVersion } from '@esql/_types/EsqlVersion'
  * Executes an ES|QL request
  * @rest_spec_name esql.query
  * @availability stack since=8.11.0 stability=experimental
+ * @availability serverless stability=experimental visibility=public
```

Comment:
what's "experimental"? should that still be kept?

Reply:
We need a "stability" value, and since ESQL is still experimental, I just used the same we have for stateful, but I don't know much more about this.